src/middlewared/middlewared/client/client.py

```diff
@@ -48,126 +44,45 @@ def wait(self, timeout=None):
 CALL_TIMEOUT = int(os.environ.get('CALL_TIMEOUT', 60))
 
 
-class ReserveFDException(Exception):
+class BindToPrivilegedPortFailed(Exception):
     pass
 
 
 class WSClient(WebSocketClient):
+
     def __init__(self, url, *args, **kwargs):
         self.client = kwargs.pop('client')
-        reserved_ports = kwargs.pop('reserved_ports')
-        reserved_ports_blacklist = kwargs.pop('reserved_ports_blacklist')
+        self.use_privileged_port = kwargs.pop('use_privileged_port, False')
+        self.privileged_ports_blacklist = kwargs.pop('reserved_ports_blacklist')
```

Comment:
Why does above have the default and this does not?

Reply:
Yep, just missed it. Thanks!